docs/Installing.asciidoc

```diff
@@ -254,6 +254,52 @@ NFS clients are where openQA workers are running. Run following command:
 mount -t nfs openQA-webUI-host:/var/lib/openqa/share /var/lib/openqa/share
 --------------------------------------------------------------------------------
 
+Auditing - tracking openQA changes
+----------------------------------
+[id="auditing"]
+
+Auditing plugin enables openQA administrators to maintain overview about what is happening with the system.
+Plugin records what event was triggered by whom, when and what the request looked like. This includes
+tracking openQA workers which are tracked under user whose API keys is worker using.
```

Comment:
the last part of this sentence sounds like it could be improved, e.g.
> This includes tracking which API keys are used by openQA workers.

If this is what you wanted to say.